### PR TITLE
Draft issuecorrection

### LINKED ISSUE #45: Remove 'Not yet in school option' a parent will not need to make request if child not in school
https://github.com/LemmaLegalConsulting/docassemble-StudentEvaluations/blob/f1d0d9a8a02cddece96fbaf68bee69955889efaf/docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml#L295-L300C11

### PR DESCRIPTION
**Pull request by CWEmrich**
Fixed #45, #46, #43, #32, #26, #28,

### CODE REVIEW

**Review by miabonardi — CHANGES_REQUESTED** [review]

File: docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_request.yml [hunk 1]
```diff
@@ -147,9 +145,13 @@ fields:
...
+      The school must also do assessments in areas that may be related to a disability.  For example, if it is hard for your child to communicate, then the school is required to conduct a speech and language evaluation and maybe an Augmentative and Alternative Communication (AAC) evaluation.  If your child has more trouble climbing the stairs at school than other students their age, the school may be required to do a physical therapy evaluation.  It is different for every student.
+      You can request specific additional evaluations if you think the school needs to evaluate your child in areas the school has not suggested.
+      It is also ok if you do not have any specific evaluation you want to request for you child.   Someone from your child’s school should contact you when they receive your request letter to discuss your concerns.  Then you can give them information that will help the school decide which evaluations your child needs.
+      
+ 
+
       
       ${ collapse_template(evaluation_definitions) }    
   - Do you want to ask for specific evaluations?: ask_specific_evaluations
```
[the remark this answers is not in the document]
Comment (CWEmrich) [reply docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_request.yml:157]: Test

**Review by miabonardi — CHANGES_REQUESTED** [review]

File: docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_request.yml [hunk 2]
```diff
@@ -160,18 +162,18 @@ fields:
     datatype: checkboxes
     choices:
```
[the remark this answers is not in the document]
Comment (CWEmrich) [reply docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_request.yml:155]: Missing info for:
(i) Social pragmatic language
(ii) Functional behavior assessment
(iii) Psychological assessment

Maybe reach out to Diana for this info?

File: docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml [hunk 3]
```diff
@@ -137,9 +139,6 @@ subquestion: |
 
   **IMPORTANT!** If you bring it to the school be sure print a copy for yourself. 
   If you can’t print a copy for yourself, when you bring the letter to the school, ask them to make a copy for you to keep for your records.
-terms:
```
Comment (miabonardi) [inline docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml:140]: Removing this will cause an on screen error where {evaluation} will no longer be defined. Why was this changed? I don't see this requested change from Diana.

File: docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml [hunk 4]
```diff
@@ -148,8 +147,6 @@ yesno: is_child_in_school
 ---
 id: before you start screen child not in school
 field: before_you_start_screen_seen
-question: |
```
Comment (miabonardi) [inline docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml:151]: removing this will cause an error. Is this interview running?

File: docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_request.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (miabonardi) [reply docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_request.yml:149]: This is still outstanding

File: docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_request.yml [hunk 5]
```diff
@@ -316,6 +318,10 @@ subquestion: |
   * This letter is for a child who is already enrolled in school.
   * You need to know which evaluations you want to ask for.
   
+  For more information on requesting an evaluation, please watch this short video
```
Comment (miabonardi) [inline docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_request.yml:333]: Delete "please"

File: docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_request.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (miabonardi) [reply docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_request.yml:155]: These help button choices do not appear to be updated

**Review by miabonardi — CHANGES_REQUESTED** [review]

**Review by miabonardi — CHANGES_REQUESTED** [review]
